logs/build.go

```diff
@@ -24,6 +25,12 @@ func (cmd *buildCmd) Run(ctx context.Context, client *api.Client) error {
 		if err := client.Get(ctx, api.NamespacedName(cmd.Name, client.Project), build); err != nil {
 			return err
 		}
+		if cmd.Since > 0 && time.Since(build.CreationTimestamp.Time) > cmd.Since {
```

Comment:
Not sure I fully understand, but this path is only taken if the user specified logs for a specific build, right? Shouldn't the since parameter then be ignored as the user clearly indicated that he wants the logs of a specific build?

Reply:
Yeah I changed this to be less confusing. Instead of using `cmd.Since` (which defaults to `logRetention` if not set) I just use `logRetention` directly. This is just so the error makes a bit more sense for the builds, without this, users would just get the generic `the logs requested exceed the retention period` error.